pkgs/development/libraries/libblockdev/default.nix

```diff
@@ -34,8 +27,8 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    glib udev kmod parted gptfdisk cryptsetup lvm2 dmraid util-linux libbytesize
-    libndctl nss volume_key libyaml
+    e2fsprogs glib udev keyutils kmod parted gptfdisk cryptsetup lvm2 dmraid util-linux libbytesize
```

Comment:
Reading [NEWS](https://github.com/storaged-project/libblockdev/releases/tag/3.0-1) and [`configure.ac` diff](https://github.com/storaged-project/libblockdev/compare/2.28-1...3.0-1#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810), `dmraid` can likely be removed, other changes of libblockdev look okay.

Reply:
Removed.